MGSwipeTableCell/MGSwipeTableCell.m

```diff
@@ -831,9 +831,12 @@ -(void) refreshContentView
     CGFloat currentOffset = _swipeOffset;
     BOOL prevValue = _triggerStateChanges;
     _triggerStateChanges = NO;
-    self.swipeOffset = 0;
-    self.swipeOffset = currentOffset;
-    _triggerStateChanges = prevValue;
+    __block MGSwipeTableCell *blockSelf = self;
```

Comment:
using the `__block` here is effectively the same as just using the original `self` reference as it creates a strong reference when captured below in the block. It would be preferred to use `__weak`, as it will create an auto-niling weak reference see (https://developer.apple.com/library/content/releasenotes/ObjectiveC/RN-TransitioningToARC/Introduction/Introduction.html) for more info